docs/packages/pkg/Boost.rst

```diff
@@ -88,6 +88,52 @@ config file (``boost/config/user.hpp``):
     #define BOOST_MPL_CFG_NO_PREPROCESSED_HEADERS
     #define BOOST_MPL_LIMIT_LIST_SIZE 3
 
+- Option ``REQUESTED_PYTHON_VERSION=<VALUE>``, if the required components of the python library
+  arguments are located, will result in ``user_config.jam`` being appended with the following line:
+
+.. code-block:: none
+
+  "using python  : <requested_version_number> : <path to python executable> :
+  <path to python include directory> : <path to directory containing the python library> ;\n".
```

Comment:
I guess quotes and `\n` is redundant here.
```
using python : <requested_version_number> : <path to python executable> :
<path to python include directory> : <path to directory containing the python library> ;
```

Reply:
Correct, it is redundant for the docs, I'll get rid of it.